ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -868,7 +869,10 @@ gst_tensor_src_tizensensor_start (GstBaseSrc * src)
 
   /* 3. Fire it up! */
   self->running = TRUE;
-  sensor_listener_start (self->listener);
+  if (sensor_listener_start (self->listener) != 0) {
+    retval = FALSE;
```

Comment:
In this case, I think `self->running = FALSE` is required.

Reply:
You're right. I'll add it.

Reply:
BTW, does `sensor_listener_start` require `self->running = TRUE`?
Otherwise, how about just moving `self->runing = TRUE` to the line 877 for code clean?

Reply:
Yeah just moving the line would be neater.